src/controls/IRoomControllerV2.ts

```diff
@@ -0,0 +1,380 @@
+import { CurrentStateValue, OldStateValue } from '../main';
+import { Control } from '../structure-file';
+import { ControlBase, ControlType } from './control-base';
+export class IRoomControllerV2 extends ControlBase {
+    private uuid = '';
+
+    async loadAsync(type: ControlType, uuid: string, control: Control): Promise<void> {
+        this.uuid = uuid;
+
+        await this.updateObjectAsync(uuid, {
+            type: type,
+            common: {
+                name: control.name,
+                role: 'thermo',
+            },
+            native: { control: control as any },
+        });
+
+        // TODO: other details not implemented - needed to get temperature unit for example (C/F).
```

Comment:
Please assume °C for now and add `unit: '°C'` to all relevant states, but leave the TODO here as we should find a way to know it from the general configuration.

Reply:
I do not agree. If we do not handle `C` or `F` properly then omitting them and letting the user assume the unit is what they have configured their system in seems the more logical choice. If you don't specify a unit then the unit cannot be incorrect. If we force `C` then our friends in the USA will complain.

Moreover, when I look at other temperature states from Loxone (1-wire sensors for example) then they have no units at all populated in IOB so I don't see an issue.